libs/ledger-live-common/src/families/cosmos/banner.test.ts

```diff
@@ -4,16 +4,18 @@ import * as logic from "./logic";
 import type { CosmosAccount, CosmosValidatorItem } from "./types";
 import data from "./preloadedData.mock";
 import cryptoFactory from "./chain/chain";
-import defaultConfig from "../../config/defaultConfig";
 import { BigNumber } from "bignumber.js";
+import { LiveConfig } from "@ledgerhq/live-config/LiveConfig";
+import liveConfig from "../../config/sharedConfig";
 
 jest.mock("./js-prepareTransaction", () => ({
   calculateFees: jest.fn(() => Promise.resolve({})),
 }));
 
 jest.mock("./chain/chain");
 
-const LEDGER_VALIDATOR_ADDRESS = defaultConfig.config.cosmos.cosmos.ledgerValidator;
+LiveConfig.setConfig(liveConfig);
+const LEDGER_VALIDATOR_ADDRESS = LiveConfig.getValueByKey("cosmos").ledgerValidator;
```

Comment:
No, 
```
LiveConfig.getValueByKey("cosmos").ledgerValidator 
```
is correct as I didn't put the family in a separate level.
I make this choice because if I put the family "cosmos" as a whole config. The config json object will be very huge and hard to maintain.

Reply:
How do we access injective ledgerValidator for example ?

Reply:
`LiveConfig.getValueByKey("injective").ledgerValidator` should work as expected from what I see here: https://github.com/LedgerHQ/ledger-live/pull/5763/files#diff-72956fd1d348841861042244e5d79c31714a02dbf1b3a3edaf9413faa60095c0R112

Reply:
yes, 
```LiveConfig.getValueByKey("injective").ledgerValidator```

Reply:
Add prefix for injective
e.g.
config_currency_injective

Reply:
fixed